javascript/new-src/src/css/style.css

```diff
@@ -217,3 +217,28 @@ div#header-universe > div{
 .settings, .input-group, .save-settings {
  margin-top: 5px;
 }
+
+/* Custom checkboxes for plugins page */
```

Comment:
The cursor is a bit funny. On my chrome it's a hand icon for the entire <tr> apart from the checkbox. I think we should have the standard pointer icon unless it moves over the Info icon or Enable checkbox, then it should be the hand cursor.